### PR TITLE
Convert Dragon Form effects to remaster rules

### PR DESCRIPTION
**Pull request by avagdumortesin**
Closes https://github.com/foundryvtt/pf2e/issues/11474.

### CODE REVIEW

**Comment by avagdumortesin** [comment]
Forgot that the UUID changes when the spell effects are compiled into the system; will recreate the links once this is merged and submit a follow-up PR for the actual spell updates.

**Comment by simonward** [comment]
> Forgot that the UUID changes when the spell effects are compiled into the system; will recreate the links once this is merged and submit a follow-up PR for the actual spell updates.

The Druid feat "Dragon Shape" will also need to be updated.

**Comment by avagdumortesin** [comment]
> > Forgot that the UUID changes when the spell effects are compiled into the system; will recreate the links once this is merged and submit a follow-up PR for the actual spell updates.
> 
> The Druid feat "Dragon Shape" will also need to be updated.

Thanks; I'll add that in once this is merged.

**Comment by stwlam** [comment]
Why are these all new effects? Are the old ones still present?

**Comment by avagdumortesin** [comment]
Yes, all of the old ones are still present; should they be removed?

**Comment by stwlam** [comment]
They should, since the new version of the spell doesn't have them. Reuse the IDs from those.

**Comment by avagdumortesin** [comment]
Ok; I'll resubmit after I get some food.

File: packs/spell-effects/spell-effect-dragon-form-mirage.json [hunk 1]
```diff
@@ -0,0 +1,230 @@
+{
+    "_id": "nWEx5kpkE8YlBZvy",
```
Comment (TikaelSol) [inline packs/spell-effects/spell-effect-dragon-form-mirage.json:2]: This id looks new, and we have 3 effects being deleted that don't have replacement effects?

Comment (avagdumortesin) [reply packs/spell-effects/spell-effect-dragon-form-mirage.json:2]: That ID is from `spell-effect-dragon-form-green`, which is one of the "deleted" files.
There were 10 dragon forms on the spell from the CRB (black, blue, brass, bronze, copper, gold, green, red, silver, white); The PC1 version only has 8 (adamantine, conspirator, diabolic, empyreal, fortune, horned, mirage, omen). I deleted silver and white, and reused the IDs from the others in alphabetical order.

Comment (avagdumortesin) [reply packs/spell-effects/spell-effect-dragon-form-mirage.json:2]: I'm not sure why Github likes to say that different files were modified than were sometimes. It should be:
Black > Adamantine
Blue > Conspirator
Brass > Diabolic
Bronze > Empyreal
Copper > Fortune
Gold > Horned
Green > Mirage
Red > Omen

**Pull request merged**